modules/jaeger-config-sampling.adoc

```diff
@@ -0,0 +1,72 @@
+////
+This REFERENCE module included in the following assemblies:
+-rhbjaeger-deploy.adoc
+////
+
+[id="jaeger-config-sampling_{context}"]
+= Jaeger sampling configuration options
+
+Note that while all traces are generated, only a few are sampled.  Sampling a trace marks the trace for further processing and storage.
+
+[NOTE]
+====
+This is not relevant if a trace was started by the Istio proxy as the sampling decision is made there. The Jaeger sampling decision is only relevant when the trace is started by an application using the Jaeger tracer.
+====
+
+The operator can be used to define sampling strategies that will be supplied to tracers that have been configured to use a remote sampler.
```

Comment:
I think this should be the first sentence after the h1. The note and the sentence that starts with "note that" should go after the when a service receives paragraph.

I also think the sentence shouldn't start with note that, especially if there is a note after it. I think it can start "While all traces..."